test/resources/recipe_source.py

```diff
@@ -53,21 +53,16 @@ def get(self):
         url = "http://api.yummly.com/v1/api{}".format(query_url)
 
         # setup and make request
-        req_obj = HttpRequestFactory.create('requests')
-        headers = {
-            'content-type': 'application/json',
-            'X-Yummly-App-ID': '9cce27e7',
-            'X-Yummly-App-Key': 'b13c741344519e5f89cb0edb7e8043f6'
-        }
+        req_obj = self.req_handler
         req_handler = RequestHandler(req_obj, redis_cache)
         res_tuple = req_handler.get(url, headers)
```

Comment:
`global name 'headers' is not defined`. Do not bother passing the second argument headers. pass in None.
Headers will be set by the base.py already.